score/liquid_icx/liquid_icx.py

```diff
@@ -344,9 +342,7 @@ def _endDistribution(self):
         The function sets the following VarDB to the default state and updates the total supply of LICX.
         """
 
-        self._total_supply.set(self.totalSupply() +
-                               self._rewards.get() +
-                               self._new_unlocked_total.get() -
+        self._total_supply.set(self.totalSupply() + self._rewards.get() + self._new_unlocked_total.get() -
```

Comment:
https://realpython.com/python-pep8/#maximum-line-length-and-line-breaking
I think this would make sense